systemd/kafka.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Apache Kafka server (broker)
+Documentation=http://kafka.apache.org/documentation.html
+Requires=network.target remote-fs.target
+After=network.target remote-fs.target
+
+[Service]
+Type=simple
+User=kafka
+Group=kafka
+RuntimeDirectory=kafka
+PIDFile=/run/kafka/kafka10-pt.pid
+Environment=KAFKA_HOME=/usr/lib/kafka CONFIG_HOME=/etc/kafka
+ExecStart=/usr/lib/kafka/bin/kafka-server-start.sh /etc/kafka/server.properties
+ExecStop=/usr/lib/kafka/bin/kafka-server-stop.sh
```

Comment:
For the ZK systemd script, I ended up adding [this](https://github.com/papertrail/salt-states/blob/261b85fcac6b930b841337db4c05437de2c60fb1/states/role_zookeeper/templates/zookeeper.service.j2#L9-L10):
```
Restart=always
RestartSec=3
```
so it'll get restarted if it crashes. I think we're doing that now with `god` and it's all good. Do we want that with Kafka as well? Could see args on both sides so just raising the question more than anything.